kubelet/datadog_checks/kubelet/common.py

```diff
@@ -91,6 +91,17 @@ def replace_container_rt_prefix(cid):
     return cid
 
 
+def get_container_label(labels, l_name):
+    """
+    Iter on all labels to find the label.name equal to the l_name
+    :param labels: list of labels
+    :param l_name: str
+    :return: str or None
+    """
+    if l_name in labels:
+        return labels[l_name]
```

Comment:
```suggestion
    return labels.get(l_name, None)
```
?

Reply:
yes indeed, thanks! this was actually a copy/paste from the existing code, I just moved the function to `common.py`.

Reply:
This is actually a list (see https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=93934&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=12901f7b-620f-5414-43d1-3ae27e5163d7&l=261)

Commit reverted.